docs/sql-data-sources-text.md

```diff
@@ -38,3 +36,35 @@ Spark SQL provides `spark.read().text("file_name")` to read a file or directory
 </div>
 
 </div>
+
+## Data Source Option
+
+Data source options of CSV can be set via:
+* the `.option`/`.options` methods of
+  *  `DataFrameReader`
+  *  `DataFrameWriter`
+  *  `DataStreamReader`
+  *  `DataStreamWriter`
+
+<table class="table">
+  <tr><th><b>Property Name</b></th><th><b>Default</b></th><th><b>Meaning</b></th><th><b>Scope</b></th></tr>
+  <tr>
+    <td><code>wholetext</code></td>
+    <td>None</td>
```

Comment:
Can you fix the default value properly? You can refer the docs in Scaladoc.

Looks like JSON and other sources have the same issue. Can you fix them in another Pr too?

Reply:
Okay, I just created sub-task, SPARK-25523.

I'll handle the other pages over there.

Reply:
SPARK-25523?

Reply:
@HyukjinKwon Sorry, SPARK-35523. I fixed the link.